template/demo/api.html

```diff
@@ -0,0 +1,48 @@
+<!--
+  Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+  See LICENSE in the project root for license information.
+
+  HTML in this document is standardized and NOT to be edited.
+  All demo code should be added/edited in ./demo/demo.md
```

Comment:
should this be updated to be `All demo code should be added/edited in ./demo/apiExamples.md` ?